complete/src/main/java/com/example/restservice/dao/Queue.java

```diff
@@ -1,40 +1,40 @@
 package com.example.restservice.dao;
 
 import java.util.List;
-import javax.persistence.*;
-
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 import org.hibernate.annotations.GenericGenerator;
 
 @Entity
 @Table(name = "queue")
+@Getter
```

Comment:
Hmm I thought the Lombok changes were already done?

Reply:
This one was missed